src/script/hooks/useHideElement.ts

```diff
@@ -1,85 +0,0 @@
-/*
- * Wire
- * Copyright (C) 2021 Wire Swiss GmbH
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program. If not, see http://www.gnu.org/licenses/.
- *
- */
-
-import {useEffect} from 'react';
-import {KEY} from 'Util/KeyboardUtil';
-import useEffectRef from 'Util/useEffectRef';
-
-const hideControlsClass = 'hide-controls';
-
-const useHideElement = (timeout: number, skipClass?: string) => {
```

Comment:
This hook was used in the previous CallingUI (where controls would appear when you move the mouse and disappear when the mouse is inactive). 

With the redesign of the calling UI, this mechanism is not needed anymore and can be removed.

@V-Gira can you confirm that there is nothing that relies on the mouse behavior with the new call UI?

Reply:
talked with @V-Gira he confirmed this is not needed anymore